MapboxDirections/MBRouteOptions.swift

```diff
@@ -335,7 +335,7 @@ open class RouteOptions: NSObject, NSSecureCoding {
         assert(!queries.isEmpty, "No query")
 
         let queryComponent = queries.joined(separator: ";")
-        return "directions/v5/\(profileIdentifier.rawValue)/\(queryComponent).json"
+        return "directions/v5/mapbox/\(profileIdentifier.rawValue)/\(queryComponent).json"
```

Comment:
Could we actually pull this out into a public variable `profilePrefix`?

Reply:
Do I have to do it ?

Reply:
@RomainQuidet yes, sometimes this is not equal to `mapbox`.

Reply:
`mapbox` still needs to be pulled out into a public property.

Reply:
@bsudekum I agree

Reply:
This change should be unnecessary, because you can initialize a `ProfileIdentifier` to hold any arbitrary identifier using `ProfileIdentifier(rawValue:)`. If this is inconvenient, an application can always extend `ProfileIdentifier` with additional constants as desired.